src/presentation/api/controller/sslController.go

```diff
@@ -33,19 +34,33 @@ func GetSslPairsController(c echo.Context) error {
 	return apiHelper.ResponseWrapper(c, http.StatusOK, sslPairsList)
 }
 
-func parseVirtualHosts(vhosts []interface{}) []valueObject.Fqdn {
-	var virtualHosts []valueObject.Fqdn
+func parseVirtualHosts(vhostsBodyInput interface{}) []valueObject.Fqdn {
+	rawVhosts := []interface{}{}
 
-	for _, vhost := range vhosts {
-		vhostStr, assertOk := vhost.(string)
+	rawVhostsInterface, assertOk := vhostsBodyInput.([]interface{})
+	if !assertOk {
+		rawVhostStr, assertOk := vhostsBodyInput.(string)
 		if !assertOk {
 			panic("InvalidVirtualHosts")
 		}
 
-		virtualHosts = append(virtualHosts, valueObject.NewFqdnPanic(vhostStr))
+		rawVhostInterface := interface{}(rawVhostStr)
+		rawVhosts = append(rawVhosts, rawVhostInterface)
 	}
 
```

Comment:
Acho que ainda tá confuso esse parseamento.

O seu pseudo seria: se for string, muda pra []interface{} pra eu poder parsear normalmente pois []interface{} é o que eu espero, logo teu código deveria ficar algo como:

```
var vhosts []valueObject.Fqn

if _, isInputString := input.(string); isInputString {
  input = []interface{input}
}

inputSliceString, assertOk := input.([]string{})
if !assertOk {
[...]
```

Reply:
Ajustado.